pygmt/xarray/backend.py

```diff
@@ -71,15 +72,17 @@ class GMTBackendEntrypoint(BackendEntrypoint):
     """
 
     description = "Open raster (.grd, .nc or .tif) files in Xarray via GMT."
-    open_dataset_parameters = ("filename_or_obj", "raster_kind")
+    open_dataset_parameters = ("filename_or_obj", "raster_kind", "region")
     url = "https://pygmt.org/dev/api/generated/pygmt.GMTBackendEntrypoint.html"
 
+    @kwargs_to_strings(region="sequence")
```

Comment:
I've been think if we should avoid using the `@kwargs_to_strings` decorator in new functions/methods, and instead write a new function like `seqjoin` which does exactly the same thing.

Reply:
Probably best to open a separate issue/PR for this.